superdesk/factory/default_settings.py

```diff
@@ -359,7 +360,15 @@ def env(variable, fallback_value=None):
 SUPERDESK_TESTING = (env('SUPERDESK_TESTING', 'false').lower() == 'true')
 
 # Default TimeZone
-DEFAULT_TIMEZONE = env('DEFAULT_TIMEZONE', 'Europe/Prague')
+DEFAULT_TIMEZONE = env('DEFAULT_TIMEZONE')
+
+if DEFAULT_TIMEZONE is None:
+    try:
```

Comment:
I raise ValueError if the value is None or Empty, but this doesn't catch any other error, and this part of code must not fail. As I can't be sure of what can be raised here (and it can change with future versions), the except Exception is needed. So yes this try/except is needed.